palette/src/rgb/mod.rs

```diff
@@ -1,4 +1,65 @@
 //! RGB types, spaces and standards.
+//!
+//! # Linear And Non-linear RGB
+//!
+//! Colors in, for example, images, are often "gamma corrected", or converted
+//! using some non-linear transfer function into a format like sRGB before being
+//! stored or displayed. This is done as a compression method and to prevent
+//! banding, and is also a bit of a legacy from the ages of the CRT monitors,
+//! where the output from the electron gun was nonlinear. The problem is that
+//! these formats are *non-linear color spaces*, which means that many
+//! operations that you may want to perform on colors (addition, subtraction,
+//! multiplication, linear interpolation, etc.) will work unexpectedly when
+//! performed in such a non-linear color space. As such, the compression has to
+//! be reverted to restore linearity and make sure that many operations on the
+//! colors are accurate.
+//!
+//! But, even when colors *are* 'linear', there is yet more to explore.
+//!
+//! The most common way that colors are defined, especially for computer
+//! storage, is in terms of so-called *tristimulus values*, meaning that all
+//! colors are defined as a vector of three values which may represent any
+//! color. The reason colors can generally be stored as only a three dimensional
+//! vector, and not an *n* dimensional one, where *n* is some number of possible
+//! frequencies of light, is because our eyes contain only three types of cones.
```

Comment:
They are the inverse of each other, so I don't know if it matters that much in this case. Maybe for what's more common in the context.

Reply:
I figured in this context since wavelength is in the next line that it'd make more sense to less technically knowledgeable users. Either is fine as you said.

Reply:
Let's settle for wavelength, then. 🙂 I like to use it too and my gut feeling is that wavelength in nanometers is the most common unit when I have been doing some research into spectral topics. Either that or wavenumber, rather than frequency.